### ISSUE TITLE
script closing after a few minutes

### ISSUE THREAD

**Issue by MrDumox** [sotolko/csgo-market-sniper#7]
hello, i've set my config to monitor 7 items on 20 pages each, and i leave script for a few minutes and i see its closed every time. what's the solution?

**Comment by TIllAd** [comment]
did u get an error message?

**Comment by MrDumox** [comment]
after a few minutes i get error on steam market, and then the console and browser close [https://ibb.co/kH8vt0c](url)

**Comment by sotolko** [comment]
Maybe this is problem?
[https://store.steampowered.com/news/group/8840272/view/3638382860930126874](url)

**Comment by MrDumox** [comment]
yep, and there is nothing to do with this. maybe there is any error handling code that would work for this error https://ibb.co/4K4912m

**Comment by sotolko** [comment]
This error message means that there are no buttons to load, because there is that steam error, so page is empty basically.

**Comment by pntdwyz** [comment]
could also be that your ip gets timed out. fix for this is adding proxies to the bot

**Comment by DqRN** [comment]
Don't get me wrong, I have no idea what I am talking about, but I keep getting the same error "TypeError: cannot unpack non-iterable NoneType object"

Is it possible to implement some sort of repeating function that will continuously search for the button that sotolko mentioned was missing or whatever it is failing to find? This way, it will keep running the script until the page reloads or the steam error fixes itself, so the script can continue rather than stoping every time.

**Comment by MoodySeel** [comment]
@DqRN Can you send the url of the skin you are trying to snipe that is giving that error?

**Comment by flyingbanana777** [comment]
I have the same issue. it just closes the script after like 5-10 minutes. no logs or nothing

**Comment by DqRN** [comment]
@Plxshy doubt it's the skin that is the issue, it's an assortment of E502 L3 errors and the steam market crashing due to the current high load. That's why I've suggested some sort of function to simply refresh and wait for it to respond instead of stopping the script. 

also is there any way to bypass the market block from submitting too many requests, cause even if the script runs for more than 5 mins after 20 you get soft banned from accessing the market.

**Comment by pntdwyz** [comment]
@DqRN proxies...

**Comment by DqRN** [comment]
@Plxshy, i just hotspot my 5g to to my PC and now I have no problems. Beside the script terminating every five minutes, cause steam errors or course.
